### DISCUSSION TITLE
Principle 2: Bring term 'audit' in

### DISCUSSION THREAD

**Discussion by heubeck** [open-gitops/project#15] [Ideas] [answered]
Initially thematized [here](https://github.com/open-gitops/documents/pull/14#discussion_r676129532) it's about the `audit-proof` suggestion: Technically it means the same as the phrases with `immutable` and `versions`, but the statement is much stronger.
Industries with deployment revision logging ("audit") demands may appreciate a less technical terminology.

First thought was to change title of principle 2 as follows:
`2. Declarations Are Versioned and Immutable` -> `2. Declarations Are Versioned and Audit-Proof`
but probably it's also worth to elaborate in the comprehensive sentence?

What do you all think?

**Comment by christianh814** [comment]
My initial instinct is to have that in the description, but not in the principal itself.

**Comment by moshloop** [comment]
I think "audit" is too overloaded of a term - it can mean different things to different people, From an IT Risk Auditor's perspective they want to know that you can provide evidence of following a governance process. The rigor in the quality of evidence will differ widely from a git log that can be overwritten to a requirement for a chain of commits signed by hardware tokens, backed by a physical log printed to paper

**Comment by williamcaban** [comment]
The word "audit" might be open to interpretations based on the background and intent of the reader. With that term there I cannot but imagine an auditor trying to use this wording to justify a "finding" against a GitOps practitioner. I will try to stay away from using the term "audit" in a principle.

The intent is for "immutable versioned declarations" which by definition will allow someone to audit the difference of changes that have occur among versions and if needed, rebuild the history of what has been used and how it has evolved over time. With that, I still consider the phrase `2. Declarations are Versioned and Immutable` express the intent while avoiding unnecessary burden by practitioners when they are under an audit.

_W

**Comment by heubeck** [comment, marked as answer]
ok, the glossary already contained it, and will continue to do: https://github.com/open-gitops/documents/pull/15/files#diff-3ccada1969baf036efeb995c3b21539c68247d55453e8d748305d12f9a359b22R72

It depends on who we are targeting with the principles title/description/glossary - in general , I'm very happy with "versioned and immutable", was thinking about @schlomo's German translation initiative, that's how "revisionssicher" - literally translated "revision safe" or more common "audit proof" came into my mind.

**Reply by scottrigby** [reply]
@heubeck can we mark this as answered?

**Reply by heubeck** [reply]
yes, please.
